website/payments/models.py

```diff
@@ -88,6 +90,8 @@ def save(self, **kwargs):
         super().save(**kwargs)
 
     def clean(self):
+        from payments.services import tpay_enabled
```

Comment:
Imports should be global

Reply:
Yeah so the problem is that this results in a circular import. And now I also don't really know what is the best to fix this 🙁 I do like the fact that Payments depends on Members and Members does not depend on Payments at all, though

Reply:
Okay I have been thinking about this and I think, codequalitywise, we should do this:
- refactor `BankAccount`s and `Payment`s to use `PaymentUser`s
- refactor the `registrations`, `events` and `pizzas` app to also user `PaymentUser`s (and for `events` perhaps even create a new `EventUser`)
- #1277 can then be added to `PaymentUser` (and perhaps the `can_access_event` should be moved to `EventUser` too)

this way, we have better uncluttered code where it is clear how all apps depend on each other:

- `members` depends on Django `auth`
- `payments` (and almost all other apps) depends on `members`
- `registrations` and `events` depend on `payments`
- `pizzas` depends on `events`

and this way we will never have circular imports ever again. 

It is quite a change but I think it is one for the better. What do you think?

Reply:
@se-bastiaan what is your opinion?

Reply:
I just discussed this with Job, I think it is a good idea.